deps/wazuh_testing/wazuh_testing/tools/s3_package.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+systems = {
+    'rpm': 'rpm',
+    'deb': 'deb',
+    'windows': 'windows',
+    'macos': 'macos',
+    'solaris10': 'solaris10',
+    'solaris11': 'solaris11',
+    'rpm5': 'rpm5',
+    'wpk-linux': 'wpk-linux',
+    'wpk-windows': 'wpk-windows'
+}
+
+architectures = {
+    'i386': 'i386',
+    'sparc': 'sparc',
+    'x86_64': 'x86_64',
+    'amd64': 'amd64',
+    'arm64v8': 'arm64v8',
+    'arm32v7': 'arm32v7'
+}
+
+def get_version(version):
```

Comment:
```suggestion
def get_short_version(version):
```

Reply:
This requested change can be reviewed in the commit : [5c3a34d2757275d66bb9951667083965a0452fab](https://github.com/wazuh/wazuh-qa/commit/5c3a34d2757275d66bb9951667083965a0452fab)